src/apps/backend/app.ts

```diff
@@ -10,7 +10,7 @@ import { AccountServer } from './modules/account';
 import { ConfigService } from './modules/config';
 import { Logger, CustomLoggerTransport } from './modules/logger';
 import { PasswordResetTokenServer } from './modules/password-reset-token';
-import { TaskServer } from './modules/task';
+import VapiServer from './modules/vapi/rest-api/vapi-server';
```

Comment:
Can you please see how task module exported the server?

We do not want to app.ts to look into inside a module?